frontend/src/screens/AdminPanel.tsx

```diff
@@ -0,0 +1,112 @@
+import { useForm } from "react-hook-form";
+import styled from "styled-components";
+import { CenteredDiv, StyledButton } from "../styles";
+
+type FormInputs = {
+  eventName: string;
+  eventDescription: string;
+  eventDate: string;
+  eventTicketsNumber: number;
+};
+
+export interface Event {
+  name: string;
+  description: string;
+  date: string;
+  ticketsNumber: number;
+}
+
+function AdminPanel() {
+  const { register, handleSubmit } = useForm<FormInputs>();
+
+  const createEvent = async (event: Event) => {
+    // TODO send request
+    console.log(event);
+  };
+
+  const onSubmit = handleSubmit(
+    async ({ eventName, eventDescription, eventDate, eventTicketsNumber }) => {
+      const event: Event = {
+        name: eventName,
+        description: eventDescription,
+        date: new Date(eventDate).toISOString(),
+        ticketsNumber: eventTicketsNumber,
+      };
+
+      createEvent(event);
+    }
+  );
+
+  return (
+    <CenteredDiv>
+      <StyledForm onSubmit={onSubmit}>
+        <LabelText htmlFor="eventName">Event name</LabelText>
+        <StyledInput
+          type="text"
+          className="form-control"
+          placeholder="Event name"
+          {...register("eventName")}
+          required
+        />
+        <LabelText htmlFor="eventDescription">Event description</LabelText>
+        <StyledInput
+          className="form-control"
+          placeholder="Event description"
+          {...register("eventDescription")}
+        />
+        <LabelText htmlFor="eventDate"> Event date </LabelText>
+        <StyledInput
+          type="date"
+          className="form-control"
+          {...register("eventDate")}
+          required
+        />
+        <LabelText htmlFor="eventTicketsNumber"> Number of tickets </LabelText>
+        <StyledInput
```

Comment:
I can set a negative value here without any error message. Or it was done like that on purpose assuming that we will validate this on the contract side?

Reply:
For now, the submit button is disabled if number of tickets is negative 
```
        <StyledButton disabled={ticketsNumber <= 0} onClick={buyTicket}>
          Buy tickets
        </StyledButton>
```
We can work on legit error message later